src/main/java/frc/robot/commands/feeder/FeederGamepieceUntilLeave.java

```diff
@@ -25,7 +28,11 @@ public void end(boolean interrupted) {
 
     @Override
     public void execute() {
-        feeder.setFeederMotorSpeed(Constants.FEEDER_MOTOR_EXIT_SPEED);
+        if (ramp.getRampPos()<Constants.RAMP_POS_SHOOT_AMP) {
+            feeder.setFeederMotorSpeed(Constants.FEEDER_MOTOR_EXIT_SPEED);
+        } else {
+            feeder.setFeederMotorSpeed(Constants.FEEDER_MOTOR_EXIT_SPEED*0.7);
+        }
```

Comment:
This needs to be another constant, this only works when the original value was 1.0 (which it might not be in the future).